spec/order_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'order'
+describe Order do
+  subject(:order) { described_class.new}
+
+  it {is_expected.to respond_to :basket}
+
+  it 'is expected to initialize with and return an empty basket hash' do
+    expect(order.basket).to eq Hash.new
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)